cms/templates/product_page.html

```diff
@@ -41,9 +41,47 @@ <h1>{{ page.title }}</h1>
             {% if start_date %}
             <div class="stat-col">
               <div class="title">start date</div>
-              <div class="text">
+              <div class="date-links-text text" id="start_date">
                 <strong>{{ start_date|date:"F j, Y" }}</strong>
               </div>
+              {% if page.product.unexpired_runs|length > 1 %}
+                {% with course_runs=page.product.unexpired_runs %}
+                  <strong class="more-dates">
+                    <a tabindex="0" role="button" class="dates-tooltip" id="datesPopover" data-trigger="focus" href="#" onClick="event.preventDefault();"
+                      data-toggle="popover" data-placement="auto" title="More dates available for this course" data-html="true"
```

Comment:
Can we shorten this title to get it down to one line?

```suggestion
                      data-toggle="popover" data-placement="auto" title="More dates available" data-html="true"
```